project2/styles.css

```diff
@@ -0,0 +1,134 @@
+* {
+    box-sizing: border-box;
+    /* border: 3px solid red; */
+}
+
+h2 {}
+
+.u-button-reset {
+    background: rgba(77, 28, 255, 0.527);
+    cursor: pointer;
+    font-family: inherit;
+    font-size: inherit;
+    font-weight: inherit;
+    padding: 0;
+    text-align: inherit;
+}
+
+h1 {
+    font-family: 'Righteous', cursive;
+    font-size: 80px;
+    text-align: center;
+    margin: 50px 0;
+}
+
+h2 {
+    font-family: 'Overpass', sans-serif;
+    font-size: 20px;
+}
+
+nav {
+    margin-bottom: 60px;
+    font-family: 'Overpass', sans-serif;
+}
+
+ul {
+    list-style: none;
+    text-align: center;
+}
+
+li+li {
+    margin-left: 100px;
+}
+
+li {
+    display: inline-block;
+}
+
+li a {
+    color: whitesmoke;
+    margin-right: 4px;
+    margin-left: 4px;
+    margin-bottom: 4px;
+    padding: 5px;
+    text-decoration: none;
+}
+
+li a:hover,
+li a:focus {
+    color: hsl(194, 100%, 50%);
+}
+
+body {
+    background-color: rgb(13, 0, 88);
+    color: whitesmoke;
+    font-family: 'Overpass', sans-serif;
+    font-size: 12px;
+}
+
+.cardbox {
+    position: relative;
+    width: 640px;
+    margin-right: auto;
+    margin-left: auto;
+}
+
+i {
+    color: whitesmoke;
+}
```

Comment:
I expect you don't need this rule, as the `i` should just inherit the text color from the `body` element.